splunk-otel-android/src/main/java/com/splunk/rum/RumInitializer.java

```diff
@@ -107,11 +107,17 @@ private String detectRumVersion() {
         return "unknown";
     }
 
-    private static void recordInitializationSpan(long startTimeNanos, List<InitializationEvent> initializationEvents, Tracer tracer) {
+    private static void recordInitializationSpan(long startTimeNanos, List<InitializationEvent> initializationEvents, Tracer tracer, Config config) {
         Span span = tracer.spanBuilder("SplunkRum.initialize")
                 .setStartTimestamp(startTimeNanos, TimeUnit.NANOSECONDS)
                 .setAttribute(SplunkRum.COMPONENT_KEY, SplunkRum.COMPONENT_APPSTART)
                 .startSpan();
+
+        String configSettings = "[debug:" + config.isDebugEnabled() + "," +
+                "crashReporting:" + config.isCrashReportingEnabled() + "," +
+                "networkMonitor:" + config.isNetworkMonitorEnabled() + "]";
+        span.setAttribute("configSettings", configSettings);
```

Comment:
Nit: should we use a more otel-like attribute name?
```suggestion
        span.setAttribute("config_settings", configSettings);
```

Reply:
no idea. :) Should we add something to the gdi-spec about this?

Reply:
In the meantime, I'm happy to use snake_case on this.